docroot/themes/humsci/humsci_basic/src/scss/utilities/_general.scss

```diff
@@ -12,10 +12,67 @@
   }
 }
 
+.hb-inline,
+.hb-inline-pipe {
+  margin-bottom: 2rem;
+
+  @include hb-colorful {
+    @include grid-media-min('sm') {
+      display: flex;
+      flex-wrap: wrap;
+    }
+
+    > div {
+      margin-bottom: hb-calculate-rems(16px);
+      padding-right: hb-calculate-rems(20px);
+    }
+  }
+}
+
+.hb-inline-pipe {
+  > div {
+    padding-right: 0;
+
+    @include grid-media-min('sm') {
+      display: flex;
+    }
+
+    &::after {
+      @include grid-media-min('sm') {
```

Comment:
I like how it drops the dividers and stacks on mobile! 🏆

Reply:
all thanks to @meranicosme